common/tests/configloader_test.go

```diff
@@ -0,0 +1,134 @@
+package tests
+
+import (
+	"encoding/json"
+	"github.com/HackIllinois/api/common/configloader"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func Setup(t *testing.T) {
+	cfg_contents := map[string]interface{}{
+		"key1": "value1",
+		"key2": "value2",
+		"key3": map[string]interface{}{
+			"key4": map[string]string{
+				"key5": "value5",
+				"key6": "value6",
+			},
+			"key7": "value7",
+		},
+	}
+
+	cfg_json, err := json.Marshal(cfg_contents)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = ioutil.WriteFile("/tmp/testconfig.json", cfg_json, 0644)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func Teardown(t *testing.T) {
```

Comment:
`TearDown`?

Reply:
`Teardown` is one word.

Reply:
This is not very important, so up to you. I think [JUnit](https://junit.org/junit4/faq.html#organize_3) uses different conventions, which is why I brought it up.